readthedocs/projects/tasks/builds.py

```diff
@@ -482,7 +471,16 @@ def on_success(self, retval, task_id, args, kwargs):
         self.data.build['success'] = True
 
     def on_retry(self, exc, task_id, args, kwargs, einfo):
```

Comment:
This task could use a docstring. When in the build flow is this triggered?

Reply:
I added a small docstring explaining _when_ this celery handler is called mentioning both flows: raising a known exception to auto retry; and calling `self.retry`